code/modules/economy/EFTPOS.dm

```diff
@@ -38,7 +38,10 @@
 	R.stamped += /obj/item/stamp
 	R.overlays += stampoverlay
 	R.stamps += "<HR><i>This paper has been stamped by the EFTPOS device.</i>"
-	var/obj/item/smallDelivery/D = new(R.loc)
+	var/obj/item/smallDelivery/D = new(get_turf(loc))
+	if(istype(loc, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = loc
+		if(H.back != null) D.forceMove(H.back)
```

Comment:
```suggestion
		if(H.back)
			D.forceMove(H.back)
```
DM can do falsy checks. Meaning it can directly if something is null or not (null being seen as FALSE)
We also put a new line after every if. Even though it only has one instruction in it's scope